src/core/state.go

```diff
@@ -23,6 +23,21 @@ import (
 	"github.com/thought-machine/please/src/process"
 )
 
+type ParseMode uint8
+
+const (
+	ParseModeForSubinclude ParseMode = 1 << iota
+	ParseModeForPreload
```

Comment:
do you need `ParseModeForPreload ParseMode`? I feel I've been caught out by this before

Reply:
It seems to compile fine so I assume not?